app/index.html

```diff
@@ -84,6 +84,7 @@
     <script src="vendor/ng-http-rate-limiter/dist/ng-http-rate-limiter.js?v=$DIM_VERSION"></script>
     <script src="vendor/angular-translate/angular-translate.min.js?v=$DIM_VERSION"></script>
     <script src="vendor/idb-keyval/dist/idb-keyval-min.js?v=$DIM_VERSION"></script>
+    <script src="vendor/angular-local-storage/dist/angular-local-storage.min.js"></script>
```

Comment:
We use localstorage all over the place without this library. What does it bring over the normal localstorage API?